Teams SPFx apps in Outlook/M365 app preview (#7386)
MicrosoftDocs/msteams-docs: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

From here, you can skip ahead to [sideload your app in Teams](#sideload-your-app-in-teams) and preview your app in Outlook and Office. (The app manifest and TeamsJS API calls have already been updated for Microsoft 365.)

### SharePoint Framework (SPFx) apps

> Starting with version 1.16 (preview) of [SharePoint Framework](/sharepoint/dev/spfx/integrate-with-teams-introduction) (SPFx), Teams personal tabs built and hosted with SPFx are also supported in Outlook and Office. To update a SPFx Teams personal tab app, you'll need to first [install the latest preview version of SPFx](/sharepoint/dev/spfx/release-1.16#install-the-latest-preview-version), and then follow the steps in this article to:
> 
> 1. [Update the app manifest](#update-the-app-manifest)
> 1. [Update SDK references](#update-sdk-references)
> 
> From there, you can skip ahead to [Sideload your app in Teams](#sideload-your-app-in-teams) to preview your SPFx personal tab app running in Outlook and Office. (You can skip the sections on configuring Content Security Policy and Azure AD app registration.)

Reviewer: ```suggestion
Starting with version 1.16 (preview) of [SharePoint Framework](/sharepoint/dev/spfx/integrate-with-teams-introduction) (SPFx), Teams personal tabs built and hosted with SPFx are also supported in Outlook and Office. To update a SPFx Teams personal tab app, follow these steps:

1. [Install the latest preview version of SPFx](/sharepoint/dev/spfx/release-1.16#install-the-latest-preview-version)
1. [Update the app manifest](#update-the-app-manifest)
1. [Update SDK references](#update-sdk-references)

After updating the SDK references, [Sideload your app in Teams](#sideload-your-app-in-teams) to preview your SPFx personal tab app running in Outlook and Office.```

Author: Good suggestion--I manually incorporated it, as the suggested change has three stray tick marks (```) at the end
